css/main.css

```diff
@@ -4,19 +4,28 @@
    Author's custom styles
    ========================================================================== */
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
```

Comment:
This styling for the images looks good, and maybe down the road we can that border radius.